### ISSUE TITLE
PrivateHD not working

### ISSUE THREAD

**Issue by theCheek** [cardigann/cardigann#60]
Finally got updated to 1.8.0-9-ga1b487c but the newly added PrivateHD from #44 doesn't seem to work.

Error I'm getting is attached, or 'Form not found matching expr 'form'

![image](https://cloud.githubusercontent.com/assets/3003042/19036300/a7f33f0a-8977-11e6-9739-8a62254df420.png)

**Comment by mmarquezs** [comment]
Try this and tell me if it works: 

Click to edit it (where you put the name and password) and change the URL from https://www.privatehd.com to https://privatehd.com (basically remove the www).

**Comment by theCheek** [comment]
bingo :) Thanks a bunch

**Comment by theCheek** [comment]
oops, just tried adding it to sonarr and got this:

> System.InvalidOperationException: Sequence contains no elements
>   at System.Linq.Enumerable.First[TSource] (IEnumerable`1 source) <0x41e5efd0 + 0x00143> in <filename unknown>:0 
>   at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () <0x4230ff50 + 0x000b7> in <filename unknown>:0

**Comment by mmarquezs** [comment]
That's true it doesn't allow adding it but I don't have any clue why. Everything looked OK from Cardigann so I assumed everything was rigth I didn't try it on sonarr.
From Cardigann side it doesn't show any message when trying to add the tracker, and in sonarr side it shows that error that I don't know what it means. 

Trying myself going in http://127.0.0.1:5060/torznab/privatehd?t=search&apikey=MYAPIKEY 
it looks ok too so I'm a little lost.

**Comment by lxt2** [comment]
That error looks like it's expecting something to contain values, but there aren't any. @mmarquezs are you able to submit a PR with a fix for that url?

**Comment by mmarquezs** [comment]
I did the pull requets to change the url, https://github.com/cardigann/cardigann/pull/65 . But the other problem I'm still clueless,  I don't see anything missing or weird I'm comparing it to thepiratebay tracker output  which works and it's clearly identical it has all the attributes etc... 

The output of http://192.168.1.52:5060/torznab/privatehd/api?t=tvsearch&cat=5030,5040&extended=1&apikey=5a4f9f4d9046aa900c62c1796b020217&offset=0&limit=100 is this one:

https://gist.github.com/mmarquezs/de9883f0f45b85a599cf453d13769473

**Comment by mmarquezs** [comment]
I found the issue I'm fixing it right now plus another issue it has, this also affects the other sites I did.
The problem is that sonarr uses tvsearch and all the definitions I made none of them has it. Sonnar checked caps and after that it failed, tought I didn't see anywhere that it was required in sonarr so it might be a bug from sonarr, well nevermind I will add a pull request with the fixes for all the trackers I made.
